src/views/network/CreateL2NetworkForm.vue

```diff
@@ -164,7 +190,8 @@ export default {
       networkOfferings: [],
       networkOfferingLoading: false,
       selectedNetworkOffering: {},
-      accountVisible: this.isAdminOrDomainAdmin()
+      accountVisible: this.isAdminOrDomainAdmin(),
```

Comment:
@davidjumani did you check against legacy UI if we should show pvlan options to domain admins?